crates/gitbutler-core/src/git/commit.rs

```diff
@@ -55,8 +57,8 @@ impl<'repo> Commit<'repo> {
         self.commit.parent(n).map(Into::into).map_err(Into::into)
     }
 
-    pub fn time(&self) -> git2::Time {
-        self.commit.time()
+    pub fn time(&self) -> Duration {
```

Comment:
The `git2::Time` is clearly specified as seconds since unix epoch in UTC and a local-time offset, even though the `time()` method here doesn't make that very obvious.

However, now there is a `time()` method that returns a `Duration` which is *implicitly* relative to the UNIX epoch and implicitly UTC, and all that without a doc-string to clarify this.

I strongly recommend to undo this so people working with `commit.time()` have more type-safety.

(An improvement here might have been to let it return `chrono` times as it's then impossible to mishandle it, but that would probably have made the downstream adaptation more difficult)